docs/topics/readme/configmap-README.adoc

```diff
@@ -89,4 +89,4 @@ $ curl http://${app-name}-MY_PROJECT_NAME.LOCAL_OPENSHIFT_HOSTNAME/api/greeting?
 ----
 
 == CI/CD Deployment
-As part of the process of creating this booster, launch.openshift.io set up a project with a CI/CD deployment of this booster. You can see the status of this deployment in your Single-node OpenShift Cluster Web Console. You can also learn more about this booster and rest of the ${runtime} runtime in the link:${guideURL}[${runtime} Runtime Guide].
\ No newline at end of file
+As part of the process of creating this booster, launch.openshift.io set up a project with a CI/CD deployment of this booster. You can see the status of this deployment in your Single-node OpenShift Cluster Web Console. You can also learn more about this booster and rest of the ${runtime} runtime in the link:${guideURL}[${runtime} Runtime Guide].
```

Comment:
Shouldn't launch.openshift.io come from the variable?

Reply:
@gytis since this is a README, we're not using the standard doc-attribute variables. The variables we use here are from the associated properties files and are runtime-specific.